easybuild/easyconfigs/n/nodejs/nodejs-12.19.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'nodejs'
+version = '12.19.0'  # LTS on 2020-10-19
+local_libversion = '72'
+
+homepage = 'https://nodejs.org'
+description = """Node.js is a platform built on Chrome's JavaScript runtime
+ for easily building fast, scalable network applications. Node.js uses an
+ event-driven, non-blocking I/O model that makes it lightweight and efficient,
+ perfect for data-intensive real-time applications that run across distributed devices."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://%(name)s.org/dist/v%(version)s/']
+sources = ['node-v%(version)s.tar.gz']
+checksums = ['1279e49be60d92cf4c1a48c9d92397db4e9284a100bc47689ce7924686bbddd1']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Python', '3.8.6'),
```

Comment:
Apparently `nodejs` *still* requires `python2` at configure time... Fixed in #11679